### ISSUE TITLE
Hyper is reading `.hyper.js` 40 times a second, constantly.

### ISSUE THREAD

**Issue by stereokai** [zeit/hyper#1699]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Windows 10 14393.953
- **Hyper.app version**: 1.3.1
- **[Link for .hyper.js](https://gist.github.com/stereokai/b377492613ad7f10073ce8049d04c8fb)**
- **Relevant information from devtools** _(CMD+ALT+I on Mac OS, CTRL+SHIFT+I elsewhere)_:
```
bundle.js:1 (re)loading renderer plugins
index.html:35 total init time 739.385
bundle.js:47 ignore pty exit: session removed by user
```
- **The issue is reproducible in vanilla Hyper.app**: YES

## Issue
I used ProcessMonitor and recorded Hyper reading the `.hyper.js` settings file on a loop. Why is it doing that? Seems wasteful and unnecessary. See the Gif I attached. The recording is filtered to Hyper.js events only. Check out the counter at the bottom: 28,000 Hyper.js events in appx. 3 minutes! Seeing that this is the same loop of 4 file system events, the math is 40 reads per second! What's the point?

![2017-03-24_21-45-50](https://cloud.githubusercontent.com/assets/2685357/24313175/246b9690-10dc-11e7-8bb8-762d736f0201.gif)

**Comment by regisbsb** [comment]
It is dog poo slow on my pc as well. Dell XPS 15 16GB RAM i7 8 core M2 SSD DISK

**Comment by ppot** [comment]
Closed in #1772
